bin/jruby.sh

```diff
@@ -1,10 +1,26 @@
 #!/bin/sh
-set -u
-# shellcheck disable=1007
+
 # -----------------------------------------------------------------------------
 # jruby.sh - Start Script for the JRuby interpreter
+#
+# This script is intended to be compatible with POSIX shell as much as possible
+# modulo a few small features known to be nonstandard but present in nearly all
+# POSIX shell implementations. We tell shellcheck to treat this source as dash,
+# a version of ash that adds those features and which has been the standard
+# Debian /bin/sh since 2011.
+#
+# shellcheck shell=dash
```

Comment:
I think shellcheck directives should be the first thing after the shebang, it's confusing to have to search for them in a big comment block

Reply:
Resolved by https://github.com/jruby/jruby/pull/8656/commits/4b776f4cb6daedf0c56c93de1b399634196763de